apis/generate.go

```diff
@@ -25,7 +25,7 @@ limitations under the License.
 // Generate deepcopy methodsets and CRD manifests
 //go:generate go run -tags generate sigs.k8s.io/controller-tools/cmd/controller-gen object:headerFile=../hack/boilerplate.go.txt paths=./... crd:trivialVersions=true output:artifacts:config=../package/crds
 
-// Generate crossplane-runtime methodsets (resource.Managed, etc)
+// Generate crossplane-runtime methodsets (resource.Claim, etc)
```

Comment:
I don't think we'll be generating any `resource.Claim` methodsets here

Reply:
ok, I will fix this in next PR.